src/Entity/Oldolf/Department.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace AppBundle\Entity\Oldolf;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\Oldolf\DepartmentRepository")
+ * @ORM\Table(name="oldolf_departments")
+ *
+ * @UniqueEntity("code")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Department
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     * @ORM\GeneratedValue(strategy="IDENTITY")
```

Comment:
We can remove the `strategy` option with `IDENTITY` value. The default strategy the `AUTO` strategy, and the `AUTO` strategy for MySQL is already `IDENTITY` 

https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/basic-mapping.html#identifier-generation-strategies